src/sentry/workflow_engine/models/data_source_detector.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class DataSourceDetector(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    data_source = FlexibleForeignKey("workflow_engine.Datasource")
```

Comment:
Sometimes you use `Datasource` and in other references it is `DataSource`. Could we be consistent with the inflection?

Reply:
That's just a mistake on my part... I'm surprised this fk actually works